tools/src/h5dump/h5dump.c

```diff
@@ -891,18 +872,6 @@ parse_command_line(int argc, const char *const *argv)
                 vfd_info_g.type   = VFD_BY_NAME;
                 vfd_info_g.u.name = H5_optarg;
                 use_custom_vfd_g  = true;
-
-#ifdef H5_HAVE_ROS3_VFD
-                if (0 == strcmp(vfd_info_g.u.name, drivernames[ROS3_VFD_IDX]))
```

Comment:
Logic like this was moved down in `h5dump`, `h5ls` and `h5stat` to handle overlap between options like `-f ros3` and `--vfd-name ros3` where the former would set the pointer to the VFD info and the latter wouldn't.

Reply:
Are they really different options?